messaging/src/test/java/org/axonframework/messaging/GenericMessageTest.java

```diff
@@ -68,15 +71,22 @@ void correlationDataAddedToNewMessage() {
     }
 
     @Test
-    void messageSerialization() {
+    void messageSerialization() throws IOException{
         GenericMessage<String> message = new GenericMessage<>("payload", Collections.singletonMap("key", "value"));
         Serializer jacksonSerializer = JacksonSerializer.builder().build();
 
         SerializedObject<String> serializedPayload = message.serializePayload(jacksonSerializer, String.class);
         SerializedObject<String> serializedMetaData = message.serializeMetaData(jacksonSerializer, String.class);
 
         assertEquals("\"payload\"", serializedPayload.getData());
-        assertEquals("{\"key\":\"value\",\"foo\":\"bar\"}", serializedMetaData.getData());
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        Map<String, String> expectedMetaData = new HashMap<>();
```

Comment:
Instead of creating another `Map` object here, you could reuse the map constructed on line 75, that's directly fed into the `GenericMessage` constructor. So, to be exact, us the `Collections.singletonMap("key", "value")` to make the metadata `Map` once within this test.

Reply:
Yes, thank you for the suggestion. I've made this change within my most recent commit.